app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -205,6 +210,11 @@ public void getAllTitlesFromArtistAsync(String artist) {
         this.execute(LVL_TITLE);
```

Comment:
```mLibraryAlbumOrder``` is initialized with LibraryAlbumOrder.ALPHABET. See:
``` private LibraryAlbumOrder mLibraryAlbumOrder = LibraryAlbumOrder.ALPHABET;```

In case someone passes ```null``` as an argument for ```mLibraryAlbumOrder``` it still sorts alphabetically, because this is the default behaviour of the sql command and this evaluates to ```false```:

```query +=  LibraryAlbumOrder.RELEASE.equals(mLibraryAlbumOrder) ? "year, " : "";```

Should I still set the default ordering in the other methods mentioned below?